oas

```diff
@@ -395,39 +352,7 @@ components:
     URN:
       type: string
       pattern: "^urn:[a-z][a-z0-9_.\\-]*:[\\w-.]+"
-      description: 'Uniform resource name of a field with the syntax `urn:<service_id>:<internal_id>`. <service_id> must be the urn of a valid and active ATLAS service. Changes for each change to have historical correct linked data.'
-    Feedgroup:
-      type: object
-      properties:
-        urn:
-          $ref: '#/components/schemas/URN'
-        history_id:
-          type: string
-          description: 'uuid which will be valid also after a change'
-        date_created:
-          type: string
-          format: date-time
-        date_last_updated:
-          type: string
-          format: date-time
-        archived:
-          type: boolean
-        name:
-          type: string
-        description:
-          type: string
-      required:
-        - urn
-        - date_created
-        - date_last_updated
-        - name
-      example:
-        urn: 'urn:animal_service:FDBE6F55-6179-4B88-9177-4AFBED33ADAC'
-        date_created: '2022-03-14T00:00:00'
-        date_last_update: '2022-05-04T00:00:42'
-        archived: false
-        name: 'Highyield 1'
-        description: '1st Pen new stable left side'
+      description: 'Uniform resource name of a field with the syntax `urn:<service_id>:<internal_id>`. <service_id> must be the urn of a valid and active ATLAS service.'
     Barn:
       type: object
       properties:
```

Comment:
optional geometry is fine and useful but there should probably be a GPS location for the barn that is mandatory to track physical location of animals in general

Reply:
- Isn't GeoJSON exactly specified for that? I have no experience how such locations are usually handled but the GeoJSON "Point" tuple seams exactly what you're looking for? Or do you prefer extra longitute/latitute fields?
- Are you sure you want this information required? Not all usecases have or need this information. For example we don't need this information to feed the animals.